frameworks/elastic/README.md

```diff
@@ -26,13 +26,19 @@ DC/OS Elasticsearch provides the following features:
 
 # Quick Start
 
-1. Install an Elasticsearch cluster and log on.
+1. Install an Elasticsearch cluster with Kibana and log on to the Mesos master node.
 
   ```bash
   dcos package install --app elastic
   dcos node ssh --master-proxy --leader
   ```
 
+1. Wait until the cluster is deployed and the nodes are all running. This may take 5-10 minutes. If you try to access the cluster too soon, you may get an empty response or an authentication error like this:
```

Comment:
Might want to point to `service/elastic/v1/plans/deploy` which when `COMPLETE` indicates the cluster is fully deployed.  There should also be a CLI command `dcos elastic plan show` which will show the same information.  This way, people can know when the deployment is actually one, instead of guessing.